env.d.ts

```diff
@@ -10,4 +10,3 @@ interface ImportMetaEnv {
 interface ImportMeta {
 	readonly env: ImportMetaEnv;
 }
-
```

Comment:
Not a big deal but you added this whitespace in the previous commit and then removed it in this one.  May be worth just not adding it in the first place ;)